server/adaptors/integrations/__data__/repository/k8s/k8s-1.0.0.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "k8s",
+    "version": "1.0.0",
+    "displayName": "k8s Dashboard",
+    "description": "k8s web logs collector",
+    "license": "k8s-2.0",
+    "type": "logs-k8s",
+    "author": "OpenSearch",
+    "sourceUrl": "https://github.com/opensearch-project/dashboards-observability/tree/main/server/adaptors/integrations/__data__/repository/k8s/info",
+    "statics": {
+        "logo": {
+            "annotation": "k8s Logo",
+            "path": "logo.png"
+        },
+        "gallery": [
+            {
+                "annotation": "k8s Dashboard",
+                "path": "dashboard.png"
+            }
+        ]
+    },
+    "components": [
+        {
+            "name": "k8s",
+            "version": "1.0.0"
+        },
+        {
+            "name": "cloud",
+            "version": "1.0.0"
+        },
+        {
+            "name": "container",
+            "version": "1.0.0"
+        },
+        {
+            "name": "logs-k8s",
```

Comment:
Did we decide we wanted to use underscores for this? I know we've had a bit of inconsistency on this point

Reply:
need to revise this globally